streams/src/main/java/org/apache/kafka/streams/processor/assignment/TaskAssignmentUtils.java

```diff
@@ -450,9 +429,9 @@ public static Map<ProcessId, KafkaStreamsAssignment> optimizeRackAwareStandbyTas
 
     private static long computeTotalAssignmentCost(final Map<TaskId, Set<TaskTopicPartition>> topicPartitionsByTaskId,
                                                    final List<TaskId> taskIds,
-                                                   final List<UUID> clientIds,
-                                                   final Map<UUID, KafkaStreamsAssignment> assignmentsByUuid,
-                                                   final Map<UUID, Optional<String>> clientRacks,
+                                                   final List<ProcessId> clientIds,
+                                                   final Map<ProcessId, KafkaStreamsAssignment> assignments,
+                                                   final Map<ProcessId, KafkaStreamsState> clients,
```

Comment:
nit: can we name these `clientAssignments` and `clientStates`? And maybe also rename `clientIds` to `clientList`?

Ditto for `#buildTaskGraph`

Just helps keep track of all these similar parameters 🙂